### PR TITLE
SW-1949 Implement csv import for inventory

### PR DESCRIPTION
**Pull request by constanzauanini**
Screen Shot 2022-10-31 at 10 44 31
Screen Shot 2022-10-31 at 10 45 01

### CODE REVIEW

File: src/components/Inventory/InventoryTable.tsx [hunk 1]
```diff
@@ -73,9 +75,22 @@ export default function InventoryTable(props: InventoryTableProps): JSX.Element
     setTemporalSearchValue(value as string);
   };
 
+  const onImportSpecies = () => {
+    setImportSpeciesModalOpen(true);
+  };
+
+  const onCloseImportSpeciesModal = (completed: boolean) => {
```
Comment (charlie-tf) [inline src/components/Inventory/InventoryTable.tsx:82]: is `completed` intended to be unused?

File: src/components/common/ImportModal.tsx [hunk 2]
```diff
@@ -336,6 +338,9 @@ export default function ImportSpeciesModal(props: ImportSpeciesModalProps): JSX.
           : undefined
       }
     >
+      {children && !error && !loading && !completed && !warning && (
+        <Box sx={{ paddingBottom: 3, color: '#3A4445', textAlign: 'left' }}> {children} </Box>
```
Comment (charlie-tf) [inline src/components/common/ImportModal.tsx:342]: `#3A4445` looks like it should be `theme.palette.TwClrTxt`

File: src/components/Inventory/InventoryTable.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/components/Inventory/InventoryTable.tsx:82]: It is not used, I will remove it

**Review by charlie-tf — APPROVED** [review]
🎉

**Pull request merged**